packages/components/Input/src/InputTokens.ts

```diff
@@ -0,0 +1,86 @@
+import type { Theme } from '@fluentui-react-native/framework';
+import { globalTokens } from '@fluentui-react-native/theme-tokens';
+import type { TokenSettings } from '@fluentui-react-native/use-styling';
+
+import type { InputTokens } from './Input.types';
+
+export const defaultInputTokens: TokenSettings<InputTokens, Theme> = (t: Theme) =>
+  ({
+    iconSize: 24,
+    hasIcon: {
+      //spacingIconContent + iconSize
+      spacingAssistiveTextStart: 16 + 24,
+      spacingLabelStart: 16 + 24,
+    },
+    spacingIconContent: 16,
+    paddingHorizontal: 16,
+    spacingInputVertical: 12,
+    spacingAssistiveTextVertical: 4,
+    spacingLabelTop: 12,
+    spacingInputSecondary: 8,
+    spacingAccessoryIconStart: 8,
+    spacingAssistiveTextStart: 0,
+    spacingLabelStart: 0,
+    accessoryIconSize: 24,
```

Comment:
Do we not have any tokens defined for this values?

Reply:
We do not have tokens for icon size, updated spacing tokens